GeoHealthCheck/plugins/probe/geonode.py

```diff
@@ -0,0 +1,34 @@
+
+from urlparse import urlparse
+import requests
+
+GEONODE_OWS_API = '/api/ows_endpoints/'
+
+
+def get_ows_endpoints(base_url):
+    r = requests.get('{}{}'.format(base_url.rstrip('/'), GEONODE_OWS_API))
+    url = urlparse(base_url)
+    base_name = 'GeoNode {}: {{}}'.format(url.hostname)
+    if r.status_code != 200:
+        msg = "Errorous response from GeoNode at {}: {}".format(base_url,
+                                                                r.text)
+        raise ValueError(msg)
+
+    try:
+        data = r.json()
+    except (TypeError, ValueError,), err:
```

Comment:
This code is needed, but does not belong here: it is not a Probe but basically part of "sniffing", setting up,  a GeoNode Resource. I realize there is a Resource-Type specific code needed, but again it does not belong here. Can you move it for the time being to `healthcheck.py`?

After your PR I would like to make `Resource Type` a Plugin. A ResourceType Plugin class would implement functions like now done explicitly via if/else in `sniff_test_resource()` in `healthcheck.py`, i.e. provide a `sniff()` function, title, return Resources (like in GeoNode case or like in e.g. a WMS/CSW linked services from Metadata) default Probe etc. 

You could consider a `GeoNodeProbe` and `Checks` though, which would be Probing/Checking a GeoNode REST API.

Reply:
> This code is needed, but does not belong here: it is not a Probe but basically part of "sniffing", setting up, a GeoNode Resource. I realize there is a Resource-Type specific code needed, but again it does not belong here. Can you move it for the time being to healthcheck.py?

Could this be added to `GeoHealthCheck/plugins/user/geonode.py`? I'd prefer to keep resource-type code away from  `healthcheck.py` to avoid making one big ball of code.

> After your PR I would like to make Resource Type a Plugin. A ResourceType Plugin class would implement functions like now done explicitly via if/else in sniff_test_resource() in healthcheck.py, i.e. provide a sniff() function, title, return Resources (like in GeoNode case or like in e.g. a WMS/CSW linked services from Metadata) default Probe etc.

+1